src/Selenium2Library/keywords/_screenshot.py

```diff
@@ -17,49 +17,85 @@ def __init__(self):
-        Setting ``persist`` specifies that the given ``path`` should
-        be used for the rest of the test execution, otherwise the path will be restored
-        at the end of the currently executing scope.
+        ``path`` argument specifies the absolute path where the screenshots
+        should be written to. If the specified ``path`` does not exist,
+        it will be created. Setting ``persist`` specifies that the given
+        ``path`` should be used for the rest of the test execution, otherwise
+        the path will be restored at the end of the currently executing scope.
         """
         path = os.path.abspath(path)
         self._create_directory(path)
         if persist is False:
             self._screenshot_path_stack.append(self.screenshot_root_directory)
             # Restore after current scope ends
-            utils.events.on('scope_end', 'current', self._restore_screenshot_directory)
+            utils.events.on('scope_end', 'current',
+                            self._restore_screenshot_directory)
 
         self.screenshot_root_directory = path
 
     def capture_page_screenshot(self, filename=None):
         """Takes a screenshot of the current page and embeds it into the log.
 
-        `filename` argument specifies the name of the file to write the
-        screenshot into. If no `filename` is given, the screenshot is saved into file
-        `selenium-screenshot-<counter>.png` under the directory where
-        the Robot Framework log file is written into. The `filename` is
+        ``filename`` argument specifies the name of the file to write the
+        screenshot into. If no ``filename`` is given, the screenshot is saved
+        into file `selenium-screenshot-{index}.png` under the directory where
+        the Robot Framework log file is written into. The ``filename`` is
         also considered relative to the same directory, if it is not
         given in absolute format. If an absolute or relative path is given
         but the path does not exist it will be created.
 
-        `css` can be used to modify how the screenshot is taken. By default
-        the bakground color is changed to avoid possible problems with
-        background leaking when the page layout is somehow broken.
+        New in Selenium2Library 1.8
+
+        If ``filename`` contains {index} characters, it will be automatically
+        replaced with running index. The running index is unique for each
+        different filename. The absolute path of the saved screenshot is always
+        returned and it does not depend does the ``filename`` contain {index}.
+        See example 1 and 2 for more details.
+
+        The {index} uses Python [https://docs.python.org/2/library/string.html#string-formatting|String Formatting]
+        and it is possible to format the {index} with [https://docs.python.org/2/library/string.html#grammar-token-width|width]
+        . The example 3 uses width to format the {index} to desired length.
```

Comment:
I would prefer long links on their own lines. Alternatively you can split them from `|`. Also, no space before the `.` character.